docker-compose/otel/docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+version: '2.4'
+services:
+  otel-collector:
+    container_name: otel-collector
+    image: otel/opentelemetry-collector:0.54.0
```

Comment:
will follow up to switch this to a sourcegraph image as we get closer to making this generally available

Reply:
> will follow up to switch this to a sourcegraph image as we get closer to making this generally available

do you plan on doing this prior mergin this PR?

Reply:
This will be available after https://github.com/sourcegraph/sourcegraph/pull/39002 , I'll follow up then